src/stylesheets/main.scss

```diff
@@ -363,3 +363,21 @@ pre .language-plaintext {
   border-bottom: 1px solid $govuk-border-colour;
   background-color: $app-light-grey;
 }
+
+.app-wcag-22 {
+  margin-top: govuk-spacing(6);
+
+  // Add highlight styling for WCAG 2.2 content
+  &:target {
```

Comment:
Can we get even spacing around this guy?

<img width="197" alt="Screenshot 2023-12-07 at 09 28 00" src="https://github.com/alphagov/govuk-design-system/assets/19834460/02a67942-b2b6-450c-968c-1670942d7f76">

Reply:
Yeah I’d like that too, will do

Reply:
I think to solve it we will have to add `margin-top: 1px;` to `.app-tag`. Will it mess the vertical rhythm?
 
<img width="1008" alt="1px extra margin above tag component" src="https://github.com/alphagov/govuk-design-system/assets/56260216/7dfb44ee-b85e-4993-9ff3-06fd620988d7">